test/end2end/pub_server-0.1.4-2.json

```diff
@@ -85,6 +85,14 @@
     "isExperimentalVersion": false,
     "isPreReleaseVersion": false
   },
+  "suggestions": [
+    {
+      "code": "tagging.notCompatible",
+      "level": "hint",
+      "title": "Package is not null safe",
```

Comment:
I believe we don't want to expose this to the users, at this point, until they cannot do anything about it. The best configuration could be to make this as a dynamic check and emit only when the SDK used for analysis already supports null-safety... wdyt?